lib/improver/utilities/cube_checker.py

```diff
@@ -171,3 +171,39 @@ def find_dimension_coordinate_mismatch(
     else:
         mismatch = list(set(second_dim_names) - set(first_dim_names))
     return mismatch
+
+
+def find_percentile_coordinate(cube):
```

Comment:
You wrote basically the same lines here:https://github.com/metoppv/improver/blob/master/lib/improver/blending/weighted_blend.py#L271
I don't know if it's worth changing these to use this function at the same time? I haven't looked carefullly,  I just remembered it looked similar.

Reply:
I'll have a look at this. It would be good to use the new functionality in this module as well if possible.